### ISSUE TITLE
Error...

### ISSUE THREAD

**Issue by cowboysdude** [watson/wifi-triangulate#4]
When I run this I get:

Geocoding service returned unexpected status code: 404

Here is how I'm calling it:

        `getPos: function(url) {
    	var self = this;
    	wifi-triangulate(function (err, location) {
         if (err) throw err
         console.log(location) // => { lat: 38.0690894, lng: -122.8069356, accuracy: 42 } 
         var result = location;
         self.sendSocketNotification("POS_RESULT", result);
         });
          },`

Or can it not be used that way?  

Thanks for a great module!

**Comment by watson** [comment]
Yeah, I've been getting that too. Apparently Google removed the free API I was using. I've been trying to find an alternative all week, but so far no luck 😢 

Any help would be much appreciated!

**Comment by bencevans** [comment]
Just come across https://www.radiocells.org/geolocation may be of interest if people are still looking.
